projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-flag/components/update-flag/update-flag.component.html

```diff
@@ -1,86 +1,112 @@
+              <div id="update-flag-char-limit-info" class="govuk-hint govuk-character-count__message" aria-live="polite">
+                {{updateFlagStepEnum.COMMENT_CHARACTER_LIMIT_INFO}}
+              </div>
+            </div>
+          </div>
         </div>
-      </div>
 
-      <div class="govuk-!-margin-bottom-6">
         <div>
-          <label class="govuk-label govuk-label--s govuk-!-margin-bottom-2">
-            {{ updateFlagTitle }} status
-          </label>
-        </div>
+          <div>
+            <label class="govuk-label govuk-label--s govuk-!-margin-bottom-2">
+              {{ updateFlagTitle }} status
+            </label>
+          </div>
 
-        <div class="govuk-!-margin-bottom-4">
-          <div class="govuk-radios" data-module="govuk-radios" [id]="FLAG_STATUS_CONTROL_NAME">
-            <div class="govuk-radios__item" *ngFor="let status of caseFlagStatuses | enumDisplayDescription">
-              <input class="govuk-radios__input" type="radio"
-                     [name]="FLAG_STATUS_CONTROL_NAME"
-                     [value]="status | lowercase">
-              <label class="govuk-label govuk-radios__label" [for]="FLAG_STATUS_CONTROL_NAME">
-                {{ status }}
-              </label>
+          <div class="govuk-!-margin-bottom-4">
+            <div class="govuk-radios" data-module="govuk-radios" [id]="FLAG_STATUS_CONTROL_NAME">
+              <div class="govuk-radios__item" *ngFor="let status of caseFlagStatuses | enumDisplayDescription">
+                <input class="govuk-radios__input" type="radio"
+                       [id]="FLAG_STATUS_CONTROL_NAME + '_' + status"
+                       [name]="FLAG_STATUS_CONTROL_NAME"
+                       [formControlName]="FLAG_STATUS_CONTROL_NAME"
+                       [value]="status">
+                <label class="govuk-label govuk-radios__label" [for]="FLAG_STATUS_CONTROL_NAME + '_' + status">
+                  {{ status }}
+                </label>
+              </div>
             </div>
           </div>
-        </div>
 
-        <div>
-          <div id="update-flag-status-hint" class="govuk-hint">
-            {{updateFlagStepEnum.STATUS_HINT_TEXT}}
-          </div>
+          <div>
+            <div id="update-flag-status-hint" class="govuk-hint">
+              {{updateFlagStepEnum.STATUS_HINT_TEXT}}
+            </div>
 
-          <textarea class="govuk-textarea"
-                    [id]="FLAG_STATUS_CHANGE_REASON_CONTROL_NAME"
-                    [name]="FLAG_STATUS_CHANGE_REASON_CONTROL_NAME" [formControlName]="FLAG_STATUS_CHANGE_REASON_CONTROL_NAME"
-                    rows="5">
+            <textarea class="govuk-textarea"
+                      [id]="FLAG_STATUS_CHANGE_REASON_CONTROL_NAME"
+                      [name]="FLAG_STATUS_CHANGE_REASON_CONTROL_NAME"
+                      [formControlName]="FLAG_STATUS_CHANGE_REASON_CONTROL_NAME"
+                      rows="5">
           </textarea>
```

Comment:
@MateiHMCTS - check alignment of `</textarea>` tag... it appears not to match indentation of the opening `<textarea>` tag